### ISSUE TITLE
Setup continuous integration

### ISSUE THREAD

**Issue by michael-dean-haynie** [michael-dean-haynie/haynie-game#10]
**Acceptance Criteria**
* Opening a PR to develop should trigger the following:
  * unit tests should be run
  * linter/formatter should be run
* Any errors should prevent the merge into develop

This issue probably depends on #8 and #9 for setup of linting/formatting and unit testing respectively.
